tests/code-splitting/subtests/between-new-worker-type/webpack/webpack.config.js

```diff
@@ -11,16 +11,21 @@
  * limitations under the License.
  */
 module.exports = {
-  resolveLoader: {
-    alias: {
-      omt: 'worker-plugin/loader?esModule',
-    },
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
```

Comment:
@developit should we put this file inside a 'workers' directory? I might be reading this wrong, but is it saying "treat all JS files as if they're workers", whereas you probably wouldn't want to do that in an actual project.

Reply:
Shouldn't need to be - this just tells webpack to find `new Worker()` invocation in JS files and treat any static string 1st argument as a bundle entry for a worker context. It applies to the main thread code that would contain the worker construction callsite.

Reply:
Ohh! Yeah, the lack of docs (or docs I couldn't find) confused me here

Reply:
```suggestion
        // treat invocations/constructions of these global functions as bundle entries
        test: /\.m?js$/,
```

Reply:
yeah, it's true that adjusting this via `module.rules` is a bit weird. Since I created this PR we added options to configure the parsers globally, which might be easier to understand:

``` js
  module: {
    parser: {
      javascript: {
        // treat invocations of these global functions as bundle entries:
        worker: [
          'InterestingNewWorkerType', // Custom Worker type
          'CSS.paintWorklet.addModule', // Worklet
          '...', // keep the defaults
        ],
      },
    },
  },
```

Reply:
@sokra is it documented anywhere? Our approach here is "if it isn't documented, it doesn't exist", but it would still count as a partial pass.